api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1248,9 +1248,72 @@ const (
 // +kubebuilder:validation:Enum=Karpenter
 type Provisioner string
 
+// Configures when and how to scale down cluster nodes.
+type ScaleDownConfig struct {
+	// enabled determines whether the Cluster Autoscaler should scaled down this cluster.
+	Enabled bool `json:"enabled"`
+
+	// delayAfterAdd sets how long after scale up the scale down evaluation resumes
```

Comment:
let's document default behaviour and an example with expected format.
MaxLength seems too generous?
We also need a minLength